src/components/landing-page/join-production.tsx

```diff
@@ -119,16 +127,28 @@ export const JoinProduction = ({ preSelected }: TProps) => {
       window.localStorage?.setItem("username", payload.username);
     }
 
+    const uuid = uuidv4();
+
     dispatch({
-      type: "UPDATE_JOIN_PRODUCTION_OPTIONS",
-      payload,
-    });
-    dispatch({
-      type: "SELECT_PRODUCTION_ID",
-      payload: null,
+      type: "ADD_CALL",
+      payload: {
+        id: uuid,
+        callState: {
+          id: uuid,
+          peerConnection: null,
+          production: null,
+          reloadProductionList: false,
+          devices: null,
+          joinProductionOptions: payload,
+          mediaStreamInput: null,
+          dominantSpeaker: null,
+          audioLevelAboveThreshold: false,
+          selectedProductionId: null,
+        },
+      },
     });
     // TODO remove
-    console.log(payload);
+    console.log("PAYLOAD: ", payload);
```

Comment:
On purpose?

Reply:
Yes. This is one of the "control-console.logs" that were left on purpose.
Wasn't sure if it's ready to be removed yet, but anoyed that it was a unnamed log so just added naming to it.